OmniKit/MessageTransport/Message.swift

```diff
@@ -100,6 +100,13 @@ struct Message {
     }
 }
 
+// returns the encoded length of a message
+public func messageLength(message: [MessageBlock]) -> Int {
```

Comment:
This should be a static func on the Message class, or just replace it with an inline version as suggested below.

Reply:
Made static func on the Message class as I didn't see an inline version suggestion